temporal-property/README.md

```diff
@@ -0,0 +1,66 @@
+--- 
+layout: pattern 
+title: Temporal Property
+folder: temporal-property
+permalink: /patterns/temporal-property/
+categories:
+- creational 
+  language: en 
+  tags:
+- Data Access
+---
```

Comment:
This table is formatted incorrectly when you open the `README`.

I am also not convinced that the tag for this pattern should be Data Access.

Reply:
Thank you for pointing out the incorrectly formatted table. The reason i chose Data Access for the tag is that data is stored and then accessed based on dates.